plugins/org.wso2.developerstudio.humantaskeditor/HumanTaskEditor/index.html

```diff
@@ -0,0 +1,508 @@
+<html xmlns:ns2="http://wso2.org/ht/schema/renderings/" xmlns:ht="http://docs.oasis-open.org/ns/bpel4people/ws-humantask/200803" xmlns="http://www.w3.org/1999/xhtml">
+
+<head>
+    <META http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <title>Untitled Document</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <link href="css/main.css" rel="stylesheet">
+    <link href="css/bootstrap.min.css" rel="stylesheet">
+    <link href="css/jquery-ui.css" rel="stylesheet">
+    <script src="js/jquery-2.1.4.min.js"></script>
+    <script type="text/javascript" src="js/jquery-ui.js"></script>
+    <script src="js/bootstrap.min.js"></script>
```

Comment:
Better to load the JS scripts at the bottom of the page. This enables to load the page lot faster.